test/ReactViews/Search/SearchBoxAndResultsSpec.tsx

```diff
@@ -99,6 +99,27 @@ describe("SearchBoxAndResults", function () {
       );
     });
 
+    it("renders with an input & no SearchInDataCatalog when showSearchInCatalogLink is false", function () {
+      const searchText = "timmynook";
+      runInAction(() => {
+        viewState.searchState.locationSearchText = searchText;
+        viewState.searchState.showLocationSearchResults = true;
+        viewState.searchState.locationSearchResults = [];
+        viewState.terria.configParameters.showSearchInCatalogLink = false;
+      });
+      act(() => {
+        testRenderer = create(
+          <ThemeProvider theme={terriaTheme}>
+            <SearchBoxAndResults
+              t={() => {}}
+              terria={terria}
+              viewState={viewState}
+            />
+          </ThemeProvider>
+        );
+      });
```

Comment:
Thanks for adding specs for your change. I think `it` blocks cannot be nested. So this needs to be extracted out as a new block.